src/Unused/Types.hs

```diff
@@ -2,17 +2,49 @@ module Unused.Types
     ( TermMatch(..)
     , TermMatchSet
     , ParseResponse(..)
+    , responseFromParse
+    , withOneFile
+    , withOneOccurrence
+    , skipRailsExtras
     ) where
 
 import Text.Parsec (ParseError)
-import Data.Map.Strict as Map
+import qualified Data.Map.Strict as Map
+import Data.List (isInfixOf)
+import Unused.Util (groupBy)
 
 data TermMatch = TermMatch
     { term :: String
     , path :: String
     , occurrences :: Int
     } deriving Show
 
-type TermMatchSet = Map String [TermMatch]
+type TermMatchSet = Map.Map String [TermMatch]
+
 
 data ParseResponse = ValidParse TermMatchSet | InvalidParse ParseError
```

Comment:
This type is pretty much identical to `Either ParseError TermMatchSet`, is there really much value to it?

I ask because you could write your logic in terms of `Parser TermMatchSet` and all those `ParserResponse -> ParseReponse` functions just be `fmap`s (`Parser` is a `Functor` et al). Then at the top level, you run the parser for the `Either` result.

WDYT?